mmdet/models/dense_heads/yolox_head.py

```diff
@@ -278,16 +278,20 @@ def get_bboxes(self,
             flatten_bboxes[..., :4] /= flatten_bboxes.new_tensor(
                 scale_factors).unsqueeze(1)
 
-        result_list = []
+        dets = []
+        labels = []
         for img_id in range(len(img_metas)):
```

Comment:
Does the exported ONNX of YOLOX support batch inference and dynamic shape features?

Reply:
any feedback here?